server/controllers/businessController.js

```diff
@@ -0,0 +1,31 @@
+// const bsService = require('../services/businessServices');
+
+//TODO: any data validation would happen within these classes
+
+const getBusinessList = (request, response) => {
```

Comment:
this could be another PR but you'll probably want to include the next parameter for error handling.

Reply:
added next parameter to each method